CMakeLists.txt

```diff
@@ -154,6 +154,12 @@ cmake_dependent_option(AIE_ENABLE_PYTHON_PASSES
 cmake_dependent_option(AIE_ENABLE_XRT_PYTHON_BINDINGS
   "Enables building of python bindings to XRT runtime." ON "XRT_COREUTIL" OFF)
 
+find_library(CDO_DRIVER_LIBRARY cdo_driver
+  HINTS "${VITIS_AIETOOLS_DIR}/lib/lnx64.o"
+)
+cmake_dependent_option(AIE_ENABLE_GENERATE_CDO_DIRECT
```

Comment:
Also depends on libxaie?

Reply:
What's libxaie? After a successful compile of `aie-translate` this is all I have 

```
mlevental@mlevental-CORSAIR-ONE-PRO-a200:~/dev_projects/mlir-aie/cmake-build-debug$ ldd ./bin/aie-translate 
        linux-vdso.so.1 (0x00007ffe471bf000)
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f548f735000)
        libzstd.so.1 => /lib/x86_64-linux-gnu/libzstd.so.1 (0x00007f548cd31000)
        libtinfo.so.6 => /lib/x86_64-linux-gnu/libtinfo.so.6 (0x00007f548f703000)
        libcdo_driver.so => /media/mlevental/9b03e9bc-780d-4e14-9b46-ec8089db51a6/Xilinx/Vitis/2023.2/aietools/lib/lnx64.o/libcdo_driver.so (0x00007f548f6fd000)
        libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f548ca00000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f548cc4a000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f548c9e0000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f548c600000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f548f775000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f548f6f6000)
```